packages/edit-post/src/components/preferences-modal/test/__snapshots__/index.js.snap

```diff
@@ -32,6 +32,11 @@ exports[`EditPostPreferencesModal should match snapshot when the modal is active
                 help="Highlights the current block and fades other content."
```

Comment:
Are these snapshot tests providing any value - I thought at some point there was a move to stop adding snapshot tests?

Reply:
Here is the note I was thinking about in relation to snapshots https://github.com/WordPress/gutenberg/pull/32592

Reply:
> Are these snapshot tests providing any value

Hmmm I'm not sure in this case. 🤔 

They're only part of this PR because they're pre-existing and I updated them with the changes.

Having said that, if you think they have little value in this context I can remove them in a new PR.

Reply:
I guess in the case of these snapshot tests they are doing something other than just checking that the component today looks exactly the same as it did yesterday - it is checking if it adapts correctly for given screen sizes, which would take a bit of additional work to test from a frontend functional perspective ... so I would be inclined to leave it as-is for now given it is an area of code that doesn't get touched often.

Reply:
Thanks for taking the time to confirm @glendaviesnz !